website/content/en/docs/User Guide/Learn Cloud Operations/cloud_error_reporting.md

```diff
@@ -37,4 +38,50 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace:
 
-![image](/docs/images/user-guide/35-logdet.png)
\ No newline at end of file
+![image](/docs/images/user-guide/35-logdet.png)
+
+#### Errors Manufacturing 
+
+There are several ways in which you can experiment with Error Reporting tool and manufacture errors that will be reported and displayed in the UI. For the purpose of this demonstration we will use 2 tools that are coming with Cloud Operations Sandbox: Load Generator and SRE Recipes to simulate a situation that `Sandbox break`.
```

Comment:
I'd suggest:

"For the purpose of this demonstration, we will use Cloud Operations Sandbox's Load Generator and SRE Recipes features to simulate errors in the system"